types/time.go

```diff
@@ -2175,6 +2175,14 @@ func ExtractDurationNum(d *Duration, unit string) (int64, error) {
 		return int64(d.Hour())*10000 + int64(d.Minute())*100 + int64(d.Second()), nil
 	case "HOUR_MINUTE":
 		return int64(d.Hour())*100 + int64(d.Minute()), nil
+	case "DAY_MICROSECOND":
```

Comment:
Add some test in `ExtractDurationNum `

Reply:
`select extract(day_second from "01 12:12:12")` arrives here.